model-client/src/commonMain/kotlin/org/modelix/model/client2/ModelClientV2.kt

```diff
@@ -227,6 +231,7 @@ abstract class ModelClientV2Builder {
         return ModelClientV2(
             httpClient?.config { configureHttpClient(this) } ?: createHttpClient(),
```

Comment:
`httpClient?.config` copies the client.
See. https://api.ktor.io/ktor-client/ktor-client-core/io.ktor.client/-http-client/config.html

So even when a custom `httpClient` is set, we should probably close its copy.